test/api_callable.js

```diff
@@ -378,4 +378,35 @@ describe('retryable', function() {
       done();
     });
   });
+
+  it('reports A/B testing', function() {
```

Comment:
This is slightly misleading, but actual A/B testing isn't enabled actually -- this test case defines a mock function of building the header and makes sure that the A/B testing values are passed into the mock function.
Actual implementation in lib/grpc.js simply ignores the A/B testing values.

Having said that, it's indeed misleading. I've removed the A/B testing code. I keep this test case because it can be reused after we enable A/B testing -- but modified as `it.skip` so that it's not tested at all.